lib/espresso-runner.js

```diff
@@ -231,6 +231,7 @@ class EspressoRunner {
       'am', 'instrument',
       '-w',
       '-e', 'debug', process.env.ESPRESSO_JAVA_DEBUG === 'true',
+      '-e', 'disableAnalytics', true, // To avoid unexpected error by google analytics
```

Comment:
I assume true should be a string rather than a boolean

Reply:
It seems fine with the current way.
When I ran with this line (both `true` and `'true'`), logcat got `UsageTrackerFacilitator: Usage tracking disabled`. But without this line, nothing appeared.